Products/ZenModel/migrate/addZingConfigToZep.py

```diff
@@ -0,0 +1,395 @@
+                            )
+                        ]
+                    ),
+                    GraphConfig(
+                        description="Seconds to Forward events to Pub/Sub.",
+                        footer=False,
+                        builtin=False,
+                        graphFormat="",
+                        miny=0,
+                        base=0,
+                        graphID="processEventSeconds",
+                        name="Events to Pub/Sub Time (seconds)",
+                        graphRange=GraphRange(
+                            end="0s-ago",
+                            start="1h-ago"
+                        ),
+                        returnset="EXACT",
+                        graphType="line",
+                        yAxisLabel="Events to Pub/Sub Time (s)",
+                        datapoints=[
+                            GraphDatapoint(
+                                aggregator="avg",
+                                color="",
+                                dataFormat="",
+                                fill=False,
+                                pointID="min",
+                                legend="min",
+                                metric="zeneventserver.zing.processEvent.min",
+                                metricSource="events-to-pubsub",
+                                name="min",
+                                rate=False,
+                                pointType="line"
+                            ),
+                            GraphDatapoint(
+                                aggregator="avg",
+                                color="",
+                                dataFormat="",
+                                fill=False,
+                                pointID="max",
+                                legend="max",
+                                metric="zeneventserver.zing.processEvent.max",
+                                metricSource="events-to-pubsub",
+                                name="max",
+                                rate=False,
+                                pointType="line"
+                            ),
+                            GraphDatapoint(
+                                aggregator="avg",
+                                color="",
+                                dataFormat="",
+                                fill=False,
+                                pointID="mean",
+                                legend="mean",
+                                metric="zeneventserver.zing.processEvent.mean",
+                                metricSource="events-to-pubsub",
+                                name="mean",
+                                rate=False,
+                                pointType="line"
+                            )
+                        ]
+                    )
+                ]
+            )
+
+            if not filter(lambda x: x.ID == 'events-to-pubsub', mcs):
+                mcs.append(
```

Comment:
The other places in this file where we make additions have a log message, so seems like this if block should have one too.